src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -1051,31 +1045,263 @@ public void GenericPublicTask_03()
             }
         }
 
+        #region Missing > after generic
+
         [Fact]
-        [CompilerTrait(CompilerFeature.InitOnlySetters)]
-        public void InitAccessor()
```

Comment:
why was this test deleted.  i'll review once the original tests are restored.  Please put all new tests at the end to avoid terrible test diffs.

Reply:
I thought I had brought it back; perhaps the merge messed it up. Bringing the original tests back, they should not be deleted.

Reply:
The tests were not removed; they were only pushed below. Since this is a large collection of tests, do I place them on another file, or the very bottom of this test file? It's about 13k lines long already, and the missing > token ones are 3k lines.

Reply:
place the new tests at the bottom.

Reply:
Done, and also brought back `InitSetAccessor` which somehow vanished. Also updated from `main` to ensure that we are not running into conflicts.